Sample/SampleForageSDK/Sections/CreatePayment/CreatePaymentView.swift

```diff
@@ -235,7 +235,7 @@ class CreatePaymentView: UIView {
                 self.fundingTypeLabel.text = "fundingType=\(response.fundingType.rawValue)"
                 self.paymentMethodIdentifierLabel.text = "paymentMethodIdentifier=\(response.paymentMethodIdentifier)"
                 self.paymentIdentifierLabel.text = "paymentIdentifier=\(response.paymentIdentifier)"
-                self.merchantAccountLabel.text = "merchantAccount=\(response.merchantAccount)"
+                self.merchantIDLabel.text = "merchantID=\(response.merchantID)"
```

Comment:
I have a feeling this is selector is going to need to be adjusted in the qa-tests.

Reply:
QA tests passed; I don't think that they rely on the naming for this^